lib/rucio/api/rule.py

```diff
@@ -246,11 +248,11 @@ def update_replication_rule(rule_id, options, issuer, vo='def', session=None):
         if not has_permission(issuer=issuer, vo=vo, action='approve_rule', kwargs=kwargs, session=session):
             raise AccessDenied('Account %s can not approve/deny this replication rule.' % (issuer))
 
-        issuer = InternalAccount(issuer, vo=vo)
+        issuer_ia = InternalAccount(issuer, vo=vo)
         if options['approve']:
-            rule.approve_rule(rule_id=rule_id, approver=issuer, session=session)
+            rule.approve_rule(rule_id=rule_id, approver=issuer_ia, session=session)
         else:
-            rule.deny_rule(rule_id=rule_id, approver=issuer, reason=options.get('comment', None), session=session)
+            rule.deny_rule(rule_id=rule_id, approver=issuer_ia, reason=options.get('comment', None), session=session)
```

Comment:
Why is this better?

Reply:
because issuer is an argument of type string. Overwriting it with an InternalAccount causes linting errors.